plugin/zoom/API/MeetingSettings.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+namespace Chamilo\PluginBundle\Zoom;
+
+class MeetingSettings
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class MeetingSettings has 25 fields. Consider redesigning MeetingSettings to keep the number of fields under 15.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)